products/extensions/subscriptions/guides/request-scoping.md

```diff
@@ -0,0 +1,42 @@
+---
+nav:
+  title: Request scoping
+  position: 20
+
+---
+
+# Request scoping
+
+When you are in a subscription checkout, you are using a separate cart and context. In Storefront, there is an additional URL parameter (`subscriptionToken`) that gets resolved. In headless, there are four header parameters that need to be set namely `sw-subscription-plan`, `sw-subscription-interval`, `_subscriptionCart` and `_subscriptionContext`.
```

Comment:
The parameters with a `_` are not in the header, but are `attributes` of the request. They tell the resolvers to replace the object with the subscription ones.